extensions/strategies/forex_analytics/strategy.js

```diff
@@ -0,0 +1,87 @@
+var z = require('zero-fill')
```

Comment:
would be really awesome to get a brief commented rundown of what this strategy does.